tracer/test/benchmarks/Benchmarks.Trace/Asm/AppSecWafBenchmark.cs

```diff
@@ -19,7 +19,7 @@ namespace Benchmarks.Trace.Asm;
 public class AppSecWafBenchmark
 {
     private const int TimeoutMicroSeconds = 1_000_000;
-
+    private const int WafRuns = 1000;
```

Comment:
Is it really needed? Usually BenchmarkDotNet is smart enough to figure out the right number of iterations to get meaningful results

Reply:
Without it, we get warnings such as Iteration time is too small , please consider increasing the number of operations.. 🤔 This made it disappear... but indeed I'm not sure of the best strategy

Reply:
that was the warning: https://bartwullems.blogspot.com/2022/09/benchmarkdotnet-minimum-observed.html